recipes/emsdk/all/conandata.yml

```diff
@@ -0,0 +1,7 @@
+sources:
+  "1.39.8":
+    sha256: 37b8807cad1aa0a976bbbdee5d3c5efc03e59175efdc555721793824f8c591f4
+    url: https://github.com/emscripten-core/emsdk/archive/1.39.8.tar.gz
```

Comment:
There's a newer 1.X https://github.com/emscripten-core/emsdk/releases/tag/1.40.1 I think this can be removed

Reply:
I'm not sure how stable these releases are when trying to compile other libraries. `1.39.8` was the version that worked for me to compile Qt (2.0 didn't). I think I haven't tried with `1.40`. It is the same with the Python and Node requirements, I was expecting _any_ of them to work, but it didn't. In the end, it is an SDK 🤷 

I will add `1.40.1`, but I want to stabilize the _compilation_ first, then test some packages and then I'll remove `1.39.8`.

Reply:
Ahh that's a good reason to keep it!